src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1626,9 +1658,10 @@ describe("Shipping", () => {
             )
             const wrapper = getWrapper({
               CommerceOrder: () => UntouchedBuyOrderWithShippingQuotes,
-              Me: () => emptyTestMe,
+              Me: () => testMe,
```

Comment:
I think the previous setup wasn't right so the test started to fail after my change. This fixes it.